### ISSUE TITLE
Flags in alphabetical order?

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#58]
It seems to me to be observing a trend in our work.
While initially the flags in the Flag section were organized in order of their appearance in the code or maybe in order of their importance, more and more the order of the flags is beginning to be in alphabetical order.
For me as tester it's quite comfortable, but I don't know whether it would be so for the average user.
This is particularly relevant for the `transaction list` command because there are tons of flags there.
If you opt for alphabetical order there, for your convenience, I can say that after the `-r` flag the flags are not in alphabetical order any more.

**Comment by d5000** [comment]
The order the flags are shown in the help is exactly the order in the code. So unfortunately there's no total freedom to organize them: if I want an argument/option to be used as positional argument (i.e. without flag keyword) then I have to put it in one of the first positions

From an usability perspective this is not ideal, but there's nothing that can be done (while using Fire). I think also it is not really a big issue if we remember that non-technical users should better consult the manual.

What I anyway try to do is to put boolean and less important flags at the end.

The "almost-alphabetical" order I had in `transaction list` was due to my reorganization of these flags to ensure all have a single-letter short form. It doesn't have usability reasons, because as I wrote that would be impossible.
